packages/dso-toolkit/src/components/legend-item/legend-item.demo.ts

```diff
@@ -2,4 +2,5 @@
 export const legendItemDemoCss = `
   .symboolcode[data-symboolcode=regelingsgebied] {
     background-image: url(images/regelingsgebied.png);
+  }
```

Comment:
Opvallende fix. Dat je daar tegen aan liep!

Reply:
Sick he. Ik moest ook met die symboolcode's aan de bak, en toen zag ik dit. Gekkigheid.

Overigens moeten we voor die symboolcode ook maar eens iets mooiers optuigen